### PR TITLE
Add missing reading of expire-after-policy when creating a NoopPersister

### LINKED ISSUE #312: NoopPersister crashes if expire-after-access memory policy is set
Hello, it appears that `Store` doesn't consider expire-after-access policy when a persister isn't present. The constructor of [NoopPersister(MemoryPolicy)](https://github.com/NYTimes/Store/blob/4109df3a336d6b73d29afdd351b683f388328ae1/store/src/main/java/com/nytimes/android/external/store/util/NoopPersister.java#L26) only includes expire-after-write rule when building its `networkResponses` field, resulting in an exception.

### PR DESCRIPTION
**Pull request by saket**
Fixes #312. I couldn't figure out a way to ensure the memory policies are used while creating cache in `NoopPersister` though.

I also made small style changes. Feel free to ask me to remove them if you want.

### CODE REVIEW

**Review by ramonaharrison — APPROVED** [review]
This looks good, thank you for submitting. We'll get one more reviewer to take a look + maybe we can help figure out a way to enhance the tests to ensure the memory policies are used.

Otherwise, LGTM. Thanks!!

File: store/src/main/java/com/nytimes/android/external/store3/util/NoopPersister.java
Comment (cybo42) [inline store/src/main/java/com/nytimes/android/external/store3/util/NoopPersister.java:33]: Technically `CacheBuilder` lets you set both `expireAfterAccess` and `expireAfterWrite` although it gives preference to `expireAfterAccess`. It's a bit of an edge case, but maybe update this block to handle both being set.

**Review by cybo42 — APPROVED** [review]
LGTM

commented on an edge case, but not sure if we need to handle it or not.

File: store/src/main/java/com/nytimes/android/external/store3/util/NoopPersister.java
Comment (saket) [reply store/src/main/java/com/nytimes/android/external/store3/util/NoopPersister.java:33]: Thanks for the review! If both are set, should we throw an exception?

**Comment by saket** [comment]
Hey folks, any updates on this?

**Comment by ramonaharrison** [comment]
I think this looks good to go. I'd like to make a 3.0.1 release available first; will merge once that has completed. Thanks!

**Pull request merged**
